charts/premium/blocky/Chart.yaml

```
@@ -43,4 +43,4 @@ sources:
   - https://hub.docker.com/r/spx01/blocky
   - https://quay.io/oriedge/k8s_gateway
 type: application
-version: 16.13.9
+version: 16.14.0
```

```suggestion
version: 16.13.10
```

This is not a feature addition.